attr.c

```diff
@@ -344,6 +344,15 @@ int iio_attr_get_range(const struct iio_attr *attr, double *min, double *step, d
 	if (!attr)
 		return -EINVAL;
 
+	/* As of now there are no available attributes for buffers and the
+	 * below wrongly detects data_available as an available kind of
+	 * attribute. If we start to see more exceptions or buffers with
+	 * valid available attributes, we can think about something like
+	 * a blacklist array.
```

Comment:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=49decddd39e5f6132ccd7d9fdc3d7c470b0061bb

+Recommended replacements for 'blacklist/whitelist' are:
+    'denylist / allowlist'
+    'blocklist / passlist'